src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -133,6 +133,43 @@ lookup_intrins_info (SimdIntrinsic *intrinsics, int size, MonoMethod *cmethod)
 	return (SimdIntrinsic *)mono_binary_search (cmethod->name, intrinsics, size / sizeof (SimdIntrinsic), sizeof (SimdIntrinsic), &simd_intrinsic_info_compare_by_name);
 }
 
+static gboolean
+has_intrinsic_cattr (MonoMethod *method)
+{
+	ERROR_DECL (error);
+	MonoCustomAttrInfo *cattr;
+	gboolean res = FALSE;
+
+	cattr = mono_custom_attrs_from_method_checked (method, error);
+	mono_error_assert_ok (error);
+
+	if (cattr) {
+		for (int i = 0; i < cattr->num_attrs; ++i) {
+			MonoCustomAttrEntry *attr = &cattr->attrs [i];
+
+			g_assert (attr->ctor);
+
+			if (!strcmp (m_class_get_name_space (attr->ctor->klass), "System.Runtime.CompilerServices") &&
+				!strcmp (m_class_get_name (attr->ctor->klass), "Intrinsic")) {
+				res = TRUE;
```

Comment:
Do we need to also check if the method is defined in corelib? In coreclr, it goes like this: https://github.com/dotnet/runtime/blob/a0a5fc9771f63aa31cb45b3577a7cea31527a2fb/src/coreclr/vm/methodtablebuilder.cpp#L1474-L1483 (IsSystem() == corelib)

Reply:
This is just a debugging function which can be uncommented to catch cases where an [Intrinsic] method is not handled.